src/components/forms/controls/CheckboxGroup/CheckboxGroup.tsx

```diff
@@ -0,0 +1,29 @@
+/* Copyright (c) Fortanix, Inc.
+|* This Source Code Form is subject to the terms of the Mozilla Public License, v. 2.0. If a copy of
+|* the MPL was not distributed with this file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { classNames as cx, type ComponentProps, type ClassNameArgument } from '../../../../util/componentUtil.ts';
```

Comment:
`type ComponentProps, type ClassNameArgument` is unused in this file but imported.